linkerd/proxy/api-resolve/src/resolve.rs

```diff
@@ -112,6 +109,7 @@ where
                             .filter_map(|addr| pb::to_addr_meta(addr, &metric_labels))
                             .collect::<Vec<_>>();
                         if !addr_metas.is_empty() {
+                            debug!(endpoints = %addr_metas.len(), "Add");
```

Comment:
nit: if `len` is a usize, we shouldn't need to use `Display`. not a big deal in practice, but we could theoretically use the typed integer value for e.g. a metric of endpoint counts or something, in the future...
```suggestion
                            debug!(endpoints = addr_metas.len(), "Add");
```